jetbrains-ultimate/tst-231+/software/aws/toolkits/jetbrains/datagrip/auth/mockDataSourceSshTunnelConfiguration.kt

```diff
@@ -0,0 +1,8 @@
+// Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.datagrip.auth
+
+import com.intellij.database.dataSource.DataSourceSshTunnelConfiguration
+
+fun mockDataSourceSshTunnelConfiguration() = DataSourceSshTunnelConfiguration(true, "config", -1)
```

Comment:
Is it possible to add the boolean parameter name here?

Reply:
it's not a kotlin class so it would just be an inline comment